deployment/Makefile

```diff
@@ -0,0 +1,11 @@
+ANSIBLE = ansible-playbook -i hosts.yml
+
+ifndef PIAF_MACHINE_USER
+	PIAF_MACHINE_USER:=$$USER
+endif
+
+install:
+	@${ANSIBLE} -u ${PIAF_MACHINE_USER} install.yml
```

Comment:
I forgot what @ means here. Couldn't we simply type `${ANSIBLE}` ?

Reply:
It's a makefile syntax, it means: "don't print the command in the output" if you want to print the executed command, you can remove the @